<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindowAdministrador</class>
 <widget class="QMainWindow" name="mainWindowAdministrador">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="cursor">
   <cursorShape>PointingHandCursor</cursorShape>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Administracion de Recursos</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="widgetPrincipal">
   <widget class="QWidget" name="widgetCabecera" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>65</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>57</red>
          <green>77</green>
          <blue>76</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>57</red>
          <green>77</green>
          <blue>76</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>159</red>
          <green>158</green>
          <blue>158</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>57</red>
          <green>77</green>
          <blue>76</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>57</red>
          <green>77</green>
          <blue>76</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="widgetUsuario" native="true">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>10</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::NoContextMenu</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/iconos/Admin.png);</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>642</x>
       <y>10</y>
       <width>131</width>
       <height>46</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="labelUsuario">
        <property name="font">
         <font>
          <family>Ubuntu Condensed</family>
          <pointsize>12</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;Ubuntu Condensed&quot;;</string>
        </property>
        <property name="text">
         <string>Usuario</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="labelSalir">
        <property name="font">
         <font>
          <family>Ubuntu Condensed</family>
          <pointsize>12</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;Ubuntu Condensed&quot;;</string>
        </property>
        <property name="text">
         <string>Salir</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>0</y>
       <width>162</width>
       <height>61</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="labelColMedica">
        <property name="styleSheet">
         <string notr="true">font: 30pt &quot;Ubuntu Condensed&quot;;
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>ColMedica</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelClinicManager">
        <property name="text">
         <string>Clinic Manager</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QToolBox" name="toolBoxHerramientas">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>200</width>
      <height>480</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="currentIndex">
     <number>4</number>
    </property>
    <widget class="QWidget" name="page">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>98</width>
       <height>28</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Personal</string>
     </attribute>
     <widget class="QCommandLinkButton" name="commandLinkButtonNuevoEmpleado">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>9</y>
        <width>181</width>
        <height>36</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Nuevo Empleado</string>
      </property>
     </widget>
     <widget class="QCommandLinkButton" name="commandLinkButtonModificarEmpleado">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>51</y>
        <width>181</width>
        <height>36</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Modificar Empleado</string>
      </property>
     </widget>
     <widget class="QCommandLinkButton" name="commandLinkButtonListarEmpleados">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>93</y>
        <width>181</width>
        <height>36</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Listar Empleados</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>98</width>
       <height>28</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Infraestructura</string>
     </attribute>
     <widget class="QLabel" name="labelAreas">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>58</width>
        <height>15</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Ubuntu Condensed</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Areas</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelCamas">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>58</width>
        <height>15</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Ubuntu Condensed</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Camas</string>
      </property>
     </widget>
     <widget class="QCommandLinkButton" name="commandLinkButtonNuevaArea">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Nueva Area</string>
      </property>
     </widget>
     <widget class="QCommandLinkButton" name="commandLinkButtonModificarArea">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>70</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Modificar Area</string>
      </property>
     </widget>
     <widget class="QCommandLinkButton" name="commandLinkButtonListarAreas">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>110</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Listar Areas</string>
      </property>
     </widget>
     <widget class="QCommandLinkButton" name="commandLinkButtonListarCamas">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>260</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Listar Camas</string>
      </property>
     </widget>
     <widget class="QCommandLinkButton" name="commandLinkButtonNuevaCama">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>180</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Nueva Cama</string>
      </property>
     </widget>
     <widget class="QCommandLinkButton" name="commandLinkButtonModificarCama">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>220</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Modificar Cama</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>98</width>
       <height>28</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Medicamentos</string>
     </attribute>
     <widget class="QCommandLinkButton" name="commandLinkButtonNuevoMedicamento">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Nuevo Medicamento</string>
      </property>
     </widget>
     <widget class="QCommandLinkButton" name="commandLinkButtonModificarMedicamento">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Modificar Medicamento</string>
      </property>
     </widget>
     <widget class="QCommandLinkButton" name="commandLinkButtonListarMedicamentos">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>90</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Listar Medicamentos</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>98</width>
       <height>28</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Habilidades Enfermera</string>
     </attribute>
     <widget class="QCommandLinkButton" name="commandLinkButtonListarHabilidades">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>100</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Listar Habilidades</string>
      </property>
     </widget>
     <widget class="QCommandLinkButton" name="commandLinkButtonModificarHabilidad">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>60</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Modificar Habilidad</string>
      </property>
     </widget>
     <widget class="QCommandLinkButton" name="commandLinkButtonNuevaHabilidad">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Nueva Habilidad</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>200</width>
       <height>315</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Informes</string>
     </attribute>
     <widget class="QCommandLinkButton" name="commandLinkButtonHistoriaClinicaPaciente">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Historia Clinica Paciente</string>
      </property>
     </widget>
     <widget class="QCommandLinkButton" name="commandLinkButtonAgendaMedicoMes">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Agenda Medico Mes</string>
      </property>
     </widget>
     <widget class="QCommandLinkButton" name="commandLinkButtonCitasAtendidasMedicoMes">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>90</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Citas Atendidas Medico Mes</string>
      </property>
     </widget>
     <widget class="QCommandLinkButton" name="commandLinkButtonCostoPaciente">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>130</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Costo Paciente</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="widgetCuerpo" native="true">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>70</y>
      <width>581</width>
      <height>481</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNuevo_empleado">
   <property name="text">
    <string>Nuevo empleado</string>
   </property>
  </action>
  <action name="actionListado_Empleados">
   <property name="text">
    <string>Modificar empleado</string>
   </property>
  </action>
  <action name="actionNueva_area">
   <property name="text">
    <string>Nueva area</string>
   </property>
  </action>
  <action name="actionListar_areas">
   <property name="text">
    <string>Listar areas</string>
   </property>
  </action>
  <action name="actionNueva_cama">
   <property name="text">
    <string>Nueva cama</string>
   </property>
  </action>
  <action name="actionListar_camas">
   <property name="text">
    <string>Listar camas</string>
   </property>
  </action>
  <action name="actionNuevo_medicamento">
   <property name="text">
    <string>Nuevo medicamento</string>
   </property>
  </action>
  <action name="actionModificar_medicamento">
   <property name="text">
    <string>Modificar medicamento</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="iconos/iconos.qrc"/>
 </resources>
 <connections/>
</ui>
